consensus/istanbul/backend/handler.go

```diff
@@ -103,7 +103,7 @@ func (sb *Backend) HandleMsg(addr common.Address, msg p2p.Msg) (bool, error) {
 	}
 	//https://github.com/ConsenSys/quorum/pull/539
 	//https://github.com/ConsenSys/quorum/issues/389
-	if msg.Code == NewBlockMsg && sb.core.IsProposer() { // eth.NewBlockMsg: import cycle
+	if msg.Code == NewBlockMsg && sb.core != nil && sb.core.IsProposer() { // eth.NewBlockMsg: import cycle
```

Comment:
is there a unit test that can capture this?

Reply:
Basically `sb.core` is nil for non-validator node.

So if we have ATs with non-validator nodes (with the flag `--mine` off) then this is well covered.

Reply:
it was only a way to capture faster, as AT take more time. 

non-blocking for me, so we can go ahead and merge.